### ISSUE TITLE
About training hyper parameters on horse2zebra.

### ISSUE THREAD

**Issue by Michi-123** [junyanz/pytorch-CycleGAN-and-pix2pix#1024]
I want to know training hyper parameters on horse2zebra.

Is 
number of epochs,
lambdas,
learning rate ,
decay point and so on 
default value?

**Comment by junyanz** [comment]
The default should be good. Check out Sec 7.1 Training details in the original [paper](https://arxiv.org/pdf/1703.10593.pdf).
